haystacker-server/src/main/kotlin/net/sr89/haystacker/server/handlers/GetBackgroundTaskProgressHandler.kt

```diff
@@ -13,13 +14,23 @@ import org.http4k.core.with
 import java.util.UUID
 
 class GetBackgroundTaskProgressHandler(private val taskManager: BackgroundTaskManager): HttpHandler {
+
+    private fun parseTaskId(taskId: String): TaskId {
```

Comment:
As mentioned in my [previous comment](https://github.com/srusso/haystacker/pull/37#discussion_r584257203), could you make this method part of `TaskId`?